src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PdfMerger.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.slc.services.steps.getpdf;
+
+import org.apache.pdfbox.multipdf.PDFMergerUtility;
+import uk.gov.hmcts.reform.slc.services.servicebus.exceptions.PdfMergeException;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static org.apache.pdfbox.io.MemoryUsageSetting.setupMainMemoryOnly;
+
+public final class PdfMerger {
+
+    private PdfMerger() {
+        // utility class constructor
+    }
+
+    public static byte[] mergeDocuments(List<byte[]> documents) {
+        ByteArrayOutputStream docOutputStream = new ByteArrayOutputStream();
```

Comment:
we can avoid doing all that if there's only one element on this list

Reply:
I had thought that but when I was writing test case I realised  that if you send only one element to merger and say that is not a pdf stream then it would simply return the first element.If you pass a stream which is not pdf then you then `pdfMergerUtility.mergeDocuments(setupMainMemoryOnly());` would throw an exception.Hence I have kept it.

Reply:
I think it's safe to assume what we get from a doc-gen is a valid pdf.
Even if not, this service is supposed to merge several docs into one, not validate whether they're valid.

Also, pdf are opened to add extra page for duplex before they end up here.